next-release-problem/mdeo/nrp/src/main/java/models/nrp/fitness/MaximiseSatisfaction.xtend

```diff
@@ -47,7 +47,7 @@ class MaximiseSatisfaction  implements IGuidanceFunction {
 		
 		println("Found satisfaction: " + satisfaction)
 		
-		return -1 * satisfaction;
+		return satisfaction;
```

Comment:
This is a minimisation problem (-1 > 1 in objective space)

Reply:
Sorry, that tendency stuff confused me again. For some reason I still thought the specified tendency will take care of the negation, now.
In fact it does for OCL guidance functions, right? Were there any drawbacks in doing the same thing for Java guidance functions?